packages/botonic-plugin-contentful/bin/migrations/migrate_contents.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/zsh
+set -e
+BIN_DIR=${0:a:h}
+
+if [[ $# != 1 ]]; then
+  echo "Usage: $0 migration_file.ts"
+  echo "Example: $0  bin/migrations/content-types/text.ts"
+  echo
+  echo "Script to add new content types or fields"
+  echo "Be aware that some content-types migrations are WIP. Review before executing them"
+  echo "To populate a new space, use bin/import-contentful-models.sh"
+  exit 1
+fi
+
+
+SPACE_ID=$CONTENTFUL_TEST_SPACE_ID
+MANAGE_TOKEN=$CONTENTFUL_TEST_MANAGE_TOKEN
+
+
+SOURCE=$(dirname "$1")/$(basename "$1" .ts)
+echo "compiling migration script"
+tsc "$SOURCE.ts"
+echo "Applying migration"
+node_modules/.bin/contentful space migration --space-id="$SPACE_ID" \
+    --access-token="$MANAGE_TOKEN" --management-token="$MANAGE_TOKEN" \
+    "$SOURCE.js"
```

Comment:
How this command will execute the migration? I'm not seeing nothing beeing executed from the javascript file...

Reply:
you need to specify as argument the .ts file from folder packages/botonic-plugin-contentful/bin/migrations/content-types this will compile it and execute it

Reply:
Yes, I saw that, what I was missing was the Migration class that is from contentful package itself. 
Everything clear now!